docker/Dockerfile.tests

```
@@ -58,27 +57,78 @@ RUN apt-get update && \
         libxml2 \
         libxml2-dev \
         locales \
+        lsof \
         make \
+        maven \
+        openjdk-8-jdk \
+        openjdk-8-jre \
         openssh-client \
         perl \
         postgresql \
         postgresql-10-pgtap \
         postgresql-contrib \
         postgresql-server-dev-10 \
+        python-dev \
```

`pip2` is needed to install SIR v1.0.2

```suggestion
        python2.7-dev \
```

---

I changed this, though I'm not sure what it does - looking at https://packages.ubuntu.com/bionic/python-dev (phusion/baseimage:0.11 uses 18.04), it was for python2 and python2.7-dev was a dependency of it.